lua/terrortown/menus/gamemode/shops/base_shops.lua

```diff
@@ -70,7 +70,7 @@ function CLGAMEMODESUBMENU:Populate(parent)
 			local item = items[i]
 
 			-- Only keep ttt-equipments that are cached
-			if not item.ttt2_cached_material and not item.ttt2_cached_model then continue end
+			if not item.isEquipment then continue end
```

Comment:
this was an old hacky solution from the old TTT2 days. If the material or model is cached, then it means that the weapon is a valid equipment item. While this is true, it is not really intuitive. This also caused issues in my bodysearch because I did not want to add another caching layer if there is already one. Therefore I refactored it. I should probably update the comment through, or remove it all together.